### ISSUE TITLE
[BUG] npm error code ENOENT while installing @vue/cli

### ISSUE THREAD

**Issue by thelifeofmiwa** [npm/cli#7634]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### This issue exists in the latest npm version

- [X] I am using the latest npm

### Current Behavior

![image](https://github.com/npm/cli/assets/163559082/8702edc4-e3b2-430f-9f5f-9735964eec5c)

### Expected Behavior

Vue is complitely installed.

### Steps To Reproduce

npm install @vue/cli

### Environment

- npm: 10.8.1
- Node.js: 18.20.3
- OS Name: Windows 10 22H2 19045.4529
- System Model Name: MC Windows 10
- npm config:
```ini
; "builtin" config from C:\Users\mihan\AppData\Roaming\npm\node_modules\npm\npmrc

prefix = "C:\\Users\\mihan\\AppData\\Roaming\\npm"

; node bin location = C:\Program Files\nodejs\node.exe
; node version = v18.20.3
; npm local prefix = D:\code\hellovue
; npm version = 10.8.1
; cwd = D:\code\hellovue
; HOME = C:\Users\mihan
```

**Comment by milaninfy** [comment]
@thelifeofmiwa based on logs it looks like npm is emitting the error for not finding the file at `"C:\Program Files\Java\jdk1.8.0_291\bin"`, npm doesn't use java as far as I know, this could be related to one of the dependency expecting jdk and creating and using java internally. It would be better to create this issue with relevant project than with npm.

**Comment by thelifeofmiwa** [comment]
> @thelifeofmiwa based on logs it looks like npm is emitting the error for not finding the file at `"C:\Program Files\Java\jdk1.8.0_291\bin"`, npm doesn't use java as far as I know, this could be related to one of the dependency expecting jdk and creating and using java internally. It would be better to create this issue with relevant project than with npm.

i think it is a part of the problem. i know that npm don't use java, but still dont know why it waht to spawn something in java directory.

**Comment by thelifeofmiwa** [comment]
I solved the problem after reinstalling Windows. I have no idea why npm needed my java files. I tried to reistall NodeJS more than 20 times in diffrent variations of build,s increase installing yarn with .msi and installing npm by yarn. I cleared Path, cleared every npm and node file in system, and did all the stuff, who was in internet about this  problem. Now, the only opportunity about this - reinstall Windows.
